### ISSUE TITLE
td-agent-bit 1.2.2 centos package missing gelf output plugin

### ISSUE THREAD

**Issue by megamorf** [fluent/fluent-bit#1467]
## Bug Report

**Describe the bug**
The CentOS 7 package of fluent-bit is missing the gelf output plugin whereas the sources/docker images have it.

**To Reproduce**
- Example log message if applicable:
```
Fluent Bit v1.2.2
Copyright (C) Treasure Data

Output plugin 'gelf' cannot be loaded
Error: You must specify an output target. Aborting
```
- Steps to reproduce the problem:

Add the CentOS repository `/etc/yum.repos.d/td-agent-bit.repo`

```
[td-agent-bit]
baseurl = https://packages.fluentbit.io/centos/7
gpgcheck = 1
gpgkey = https://packages.fluentbit.io/fluentbit.key
name = TD Agent Bit
```

Then install `td-agent-bit` (version 1.2.2):

```
$ sudo yum install -y td-agent-bit
$ td-agent-bit --version
Fluent Bit v1.2.2
```

Add this output section to `/etc/td-agent-bit/td-agent-bit.conf`:

```
[OUTPUT]
    Name     gelf
    Match    *
    Host     log.evilcorp.net
    Port     12201
    Mode     udp
    # Mandatory configuration, log is the key of the message
    Gelf_Short_Message_Key log
```

Run fluent-bit

```
sudo /usr/local/bin/td-agent-bit -c /etc/td-agent-bit/td-agent-bit.conf
```

**Expected behavior**
Gelf output being sent to the remote log host.

There seem to be differences between the td-agent-bit and the docker version:

```
/usr/local/bin/td-agent-bit -h > /tmp/fluent-td.txt
docker run fluent/fluent-bit:1.2.2 /fluent-bit/bin/fluent-bit -h > /tmp/fluent-docker.txt
```

Comparison

```diff
--- a/tmp/fluent-docker.txt
+++ b/tmp/fluent-td.txt
@@ -66,7 +66,6 @@ Outputs
  tcp                   TCP Output
  td                    Treasure Data
  flowcounter           FlowCounter
[-  gelf                  GELF Output-]

Filters
  stdout                Filter events to STDOUT
@@ -81,4 +80,4 @@ Filters

Internal
 Event Loop  = epoll
 Build Flags =  FLB_HAVE_PARSER FLB_HAVE_STREAM_PROCESSOR JSMN_PARENT_LINKS JSMN_STRICT FLB_HAVE_TLS FLB_HAVE_SQLDB {+FLB_HAVE_TRACE+} FLB_HAVE_METRICS FLB_HAVE_HTTP_SERVER FLB_HAVE_FLUSH_LIBCO FLB_HAVE_SYSTEMD FLB_HAVE_FORK[-FLB_HAVE_TIMESPEC_GET-] FLB_HAVE_GMTOFF FLB_HAVE_UNIX_SOCKET {+FLB_IS_TD_AGENT+} FLB_HAVE_PROXY_GO FLB_HAVE_SYSTEM_STRPTIME FLB_HAVE_JEMALLOC JEMALLOC_MANGLE FLB_HAVE_LIBBACKTRACE FLB_HAVE_REGEX FLB_HAVE_LUAJIT FLB_HAVE_C_TLS FLB_HAVE_ACCEPT4 FLB_HAVE_INOTIFY
```

**Comment by megamorf** [comment]
If I build the rpm via https://github.com/fluent/fluent-bit-packaging the gelf output plugin is still missing. I had a look at the fluent-bit source code and gelf is skipped if zlib is not available. This dependency is installed in the official docker image but is missing from the CentOS Dockerfile in the fluent-bit-packaging repo.

I built the master branch of fluent-bit after adding the zlib package to the Dockerfile and as expected, the gelf output plugin is now avaible:

```diff
 Outputs
   azure                 Send events to Azure HTTP Event Collector
   bigquery              Send events to BigQuery via streaming insert
   counter               Records counter
   es                    Elasticsearch
   exit                  Exit after a number of flushes (test purposes)
   file                  Generate log file
   forward               Forward (Fluentd protocol)
   http                  HTTP Output
   influxdb              InfluxDB Time Series
   kafka                 Kafka
   kafka-rest            Kafka REST Proxy
   nats                  NATS Server
   null                  Throws away events
   plot                  Generate data file for GNU Plot
   splunk                Send events to Splunk HTTP Event Collector
   stackdriver           Send events to Google Stackdriver Logging
   stdout                Prints events to STDOUT
   tcp                   TCP Output
   td                    Treasure Data
   flowcounter           FlowCounter
+  gelf                  GELF Output
```

**Comment by megamorf** [comment]
Please merge the following PR to fix this:
https://github.com/fluent/fluent-bit-packaging/pull/8

**Comment by megamorf** [comment]
Bump

**Comment by megamorf** [comment]
@edsiper @nokute78

**Comment by edsiper** [comment]
thanks for reporting the problem. This is fixed in v1.3 release
